applications/vanilla/views/discussion/tag.php

```diff
@@ -0,0 +1,16 @@
+<?php if (!defined('APPLICATION')) exit(); ?>
+    <h1><?php echo $this->data('Title'); ?></h1>
+<?php
+/** @var Gdn_Form $form */
+$form = $this->Form;
+echo $form->open(['id' => 'DiscussionAddTagForm']);
+echo $form->errors();
+
+echo '<div class="Form-Tags P">';
+    // Tag text box
+    echo $this->Form->label('Tags', 'Tags');
+    echo $this->Form->textBox('Tags', array('data-tags' => ''));
+echo '</div>';
+
+?>
+<?php echo $this->Form->close('Save');
```

Comment:
Styling in the admin area versus the public-facing site is a little different. If you look at windows for other options in the same discussion menu, you'll see there's a little bit more to them.